openbas-api/src/main/java/io/openbas/rest/inject/output/InjectOutput.java

```diff
@@ -0,0 +1,85 @@
+package io.openbas.rest.inject.output;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import io.openbas.database.model.InjectorContract;
+import io.openbas.helper.InjectModelHelper;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import lombok.Data;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+@Data
+public class InjectOutput {
+
+  @JsonProperty("inject_id")
+  @NotBlank
+  private String id;
+
+  @JsonProperty("inject_title")
+  private String title;
+
+  @JsonProperty("inject_enabled")
+  private boolean enabled;
+
+  @JsonProperty("inject_exercise")
+  private String exercise;
+
+  @JsonProperty("inject_scenario")
+  private String scenario;
+
+  @JsonProperty("inject_depends_duration")
+  @NotNull
+  @Min(value = 0L, message = "The value must be positive")
```

Comment:
Since this can only be used for outputs, are validation annotation still needed ?

Reply:
This is usefull for api-types file
Besides, I forgot to update it, thank you!